desktop/src/com/mygdx/game/client/di/ComponentMessageListenerModule.java

```diff
@@ -31,29 +31,29 @@ public class ComponentMessageListenerModule {
 
   @Provides
   public ComponentMessageListener provideComponentMessageListener(
-      EntityConfigHandler entityConfigHandler,
-      SubFieldHandler subFieldHandler,
       BuildingHandler buildingHandler,
-      FieldHandler fieldHandler,
+      CanAttackHandler canAttackHandler,
       CoordinatesHandler coordinatesHandler,
-      MaterialHandler materialHandler,
       DesktopEntityConfigHandler desktopEntityConfigHandler,
+      EntityConfigHandler entityConfigHandler,
+      FieldHandler fieldHandler,
+      MaterialHandler materialHandler,
       OwnerHandler ownerHandler,
       StatsHandler statsHandler,
-      CanAttackHandler canAttackHandler,
+      SubFieldHandler subFieldHandler,
       NetworkWorldEntityMapper mapper
   ) {
     var listener = new ComponentMessageListener(mapper); // jobs
-    listener.registerHandler(EntityConfigId.class, entityConfigHandler);
-    listener.registerHandler(EntityConfigId.class, desktopEntityConfigHandler);
-    listener.registerHandler(MaterialComponent.class, materialHandler);
+    listener.registerHandler(Building.class, buildingHandler);
+    listener.registerHandler(CanAttack.class, canAttackHandler);
     listener.registerHandler(Coordinates.class, coordinatesHandler);
+    listener.registerHandler(EntityConfigId.class, desktopEntityConfigHandler);
+    listener.registerHandler(EntityConfigId.class, entityConfigHandler);
     listener.registerHandler(Field.class, fieldHandler);
-    listener.registerHandler(SubField.class, subFieldHandler);
-    listener.registerHandler(Building.class, buildingHandler);
+    listener.registerHandler(MaterialComponent.class, materialHandler);
     listener.registerHandler(Owner.class, ownerHandler);
     listener.registerHandler(Stats.class, statsHandler);
-    listener.registerHandler(CanAttack.class, canAttackHandler);
+    listener.registerHandler(SubField.class, subFieldHandler);
     return listener;
```

Comment:
może to nadal działa, ale ja bym się bał zmieniać kolejność tych handlerów

Reply:
Hmm, no działa nadal, też o tym myślałem, ale ostatecznie pomyślałem, że chyba wstawiałem tu cały czas losowo i działa, ale dla bezpieczeństwa przywrócę